### DISCUSSION TITLE
Safari IOS 15.4.1 (lasted) can't open AR mode

### DISCUSSION THREAD

**Discussion by jatumin01** [google/model-viewer#3384] [Q&A] [answered]
lasted version  src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"

my model-viewer tag use **ios-src for model.usdz**

error is "Object requires a newer vresion of IOS"

https://user-images.githubusercontent.com/29621744/164202877-3a4daf65-b9f3-4bc4-9baf-618b70df9112.mp4

![S__14188569](https://user-images.githubusercontent.com/29621744/164203078-c52c3a9b-b1ad-40a8-8b97-d166c5719cf8.jpg)

**Comment by elalish** [comment, marked as answer]
This is Apple's generic error for "couldn't parse the USDZ". Try exporting it again and ensure it's being served properly.

**Reply by jatumin01** [reply]
thank for suggestion, I have tried exporting again. and check the .usdz file  and found that the problem was caused by The server is not responding to that file. now i can fix it :))

**Comment by seadeep42** [comment]
I'd suggest checking if Reality converter is the latest version. https://developer.apple.com/augmented-reality/tools/
For me, a similar issue was resolved after the recent os update by reinstalling the latest version of reality converter

**Reply by jatumin01** [reply]
Thank for suggestion , I have tried following your suggestion But the problem is from the server side. haha

**Reply by vortice3D** [reply]
Hi @jatumin01:

At the moment, I'm experiencing this same kind of problems after updating to iOS17 (iOS16 and below work flawless). The case is that when I try to access a .reality file, the "Object requires a newer vresion of IOS." appears on-screen, but not in the case of using a USDZ.

I'm using the cosmonaut .reality file example from [Apple Developer QLAR examples page](https://developer.apple.com/augmented-reality/quick-look/), and following the guidelines my markup is:

```
<a rel="ar" href="/assets/models/CosmonautSuit_en.reality">
    <img class="image-model" src="/assets/posters/cosmonaut.png">
</a>
```

Curiously, if I don't include the rel="ar" option or drop the URL of the .reality directly to the Safari browsing editbox, the augmented reality is launched without issues, but of course showing the ugly auto-generated poster, that ruins all the UX/UI.

Any tip on all this?

Thanks for your time.
